settings.gradle

```diff
@@ -1,6 +1,15 @@
 // https://docs.gradle.com/enterprise/gradle-plugin/#gradle_6_x_and_later
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        google()
+        mavenCentral()
+    }
+}
+
 plugins {
     id "com.gradle.enterprise" version "3.3.3"
+    id "org.jetbrains.dokka" version "2.0.0" apply false
```

Comment:
Are we using dokka now, or is this for later? :)

Reply:
Yeah, I also noticed this when I was trying to fix CI. I don't think we use dokka, I'm guessing one of our dependencies is incorrectly pulling dokka. We can proceed with this for now and figure it out later.